pwiz_tools/Skyline/Properties/Resources.resx

```diff
@@ -3839,372 +3836,6 @@ Go to the Peptide Settings Prediction tab to fix.</value>
   <data name="PanoramaDownload" type="System.Resources.ResXFileRef, System.Windows.Forms">
     <value>..\Resources\PanoramaDownload.bmp;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
-  <data name="RefineDlg_RefineDlg_Precursors" xml:space="preserve">

```

Comment:
Seems like a lot of deleted strings with no clear relation to Prosit/Koina. Can you explain this? Is Nick's new tool somehow responsible?

Reply:
Probably a merge mistake. I'll try rebasing.